scripts/effects/super_buff.lua

```diff
@@ -6,10 +6,9 @@ local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
     local power = effect:getPower()
-    target:addMod(xi.mod.ATTP, power)
-    target:addMod(xi.mod.DEFP, power)
-    target:addMod(xi.mod.MATT, power)
-    target:addMod(xi.mod.MEVA, power)
+    target:addMod(xi.mod.UDMGMAGIC, -500)
+    target:addMod(xi.mod.UDMGPHYS, -5000)
+    target:addMod(xi.mod.UDMGRANGE, -5000)
```

Comment:
is that -500 correct?

Reply:
Also:
effect:addMod() removes the need to have to delete the mods in onEffectLoose

Reply:
I ended up reverting this change after double checking that, looking at captures of Nidhogg the existing stats seem accurate to me.